hook.js

```diff
@@ -217,6 +217,14 @@ function addIitm (url) {
 function createHook (meta) {
   let cachedResolve
   const iitmURL = new URL('lib/register.js', meta.url).toString()
+  let includeLibs, excludeLibs
+
+  async function initialize (options) {
+    if (options) {
+      includeLibs = options && Array.isArray(options.include) ? options.include : undefined
```

Comment:
we can remove the `option` coercion check because we do `if(options)` above.

Given that this is done in the initializer, we can also get away with some debug logging is someone passes in invalid options.

Also, what should we do with empty arrays? Ignore them?

From first thoughts:

empty include -> no files should be included
empty exclude -> same as undefined exclude